### PR TITLE
LOC7454 55

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Locations for Dabra Hallelo Enda Abuna Samuel

### CODE REVIEW

File: new/LOC7454WaddiAmbaray.xml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            <change who="DN" when="2024-09-10">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+     <place type="tabiya">
+                    <placeName/>
+                    <placeName xml:id="n1" xml:lang="ti">ማይ፡ ወዲ፡ ኣምበራይ</placeName>
+         <placeName corresp="#n1" xml:lang="en">Māy Waddi ʾAmbarāy</placeName>
```
Comment (CarstenHoffmannMarburg) [inline new/LOC7454WaddiAmbaray.xml:50]: Māy Waddi ʾĀmbarāy (?)

File: 1001-2000/LOC1121AddiAr.xml [hunk 2]
```diff
@@ -42,15 +42,25 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
          <listPlace>
-            <place type="region">
-               <placeName>ʿAddi Arbaʿǝtä</placeName>
+            <place type="warada">
+               <placeName/>
+               <placeName xml:id="n1" xml:lang="ti">ዓዲ፡ አርባዕተ፡</placeName>
+               <placeName corresp="#n1" xml:lang="en">ʿAddi ʾArbaʿǝta</placeName>
+               <placeName xml:id="n2" xml:lang="ti">ራማ፡ ዓዲ፡ አርባዕተ፡</placeName>
+               <placeName corresp="#n2" xml:lang="en">Rāmā ʿAddi ʾArbaʿǝta</placeName>
+               <note>The name change from ʿAddi ʾArbaʿǝta to Rāmā ʿAddi ʾArbaʿǝta in 2000s.</note>
```
Comment (CarstenHoffmannMarburg) [inline 1001-2000/LOC1121AddiAr.xml:57]: ʿAddi ʾArbāʿǝta (mark ā)

File: 1001-2000/LOC1121AddiAr.xml [hunk 3]
```diff
@@ -42,15 +42,25 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPlace>
-            <place type="region">
-               <placeName>ʿAddi Arbaʿǝtä</placeName>
+            <place type="warada">
+               <placeName/>
+               <placeName xml:id="n1" xml:lang="ti">ዓዲ፡ አርባዕተ፡</placeName>
+               <placeName corresp="#n1" xml:lang="en">ʿAddi ʾArbaʿǝta</placeName>
```
Comment (CarstenHoffmannMarburg) [inline 1001-2000/LOC1121AddiAr.xml:54]: ʿAddi ʾArbāʿǝta (mark ā)

File: 1001-2000/LOC1121AddiAr.xml [hunk 4]
```diff
@@ -4,7 +4,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title>ʿAddi Arbaʿǝtä</title>
+            <title>ʿAddi ʾArbaʿǝta</title>
```
Comment (CarstenHoffmannMarburg) [inline 1001-2000/LOC1121AddiAr.xml:7]: ʿAddi ʾArbāʿǝta (mark ā)

**Review by CarstenHoffmannMarburg — CHANGES_REQUESTED** [review]

File: 1001-2000/LOC1121AddiAr.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply 1001-2000/LOC1121AddiAr.xml:57]: ok

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Review by nafisa-valieva — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**